src/Maestro/Maestro.Data/Models/Build.cs

```diff
@@ -152,6 +153,20 @@ public string GitHubBranch
         /// </summary>
         public bool Stable { get; set; }
 
+        /// <summary>
+        /// JSON string containing incoherency properties & nodes for the subtree
+        /// starting at this build.
+        /// </summary>
+        [Column("Incoherencies")]
+        public string BuildIncoherenciesString { get; set; }
```

Comment:
I wonder whether we could avoid a json blob here and instead just use a table. If the interesting fields are just Name, Version, Repo, and Sha, could we just have a table with rows of Name, Version, Repo, Sha, and BuildId?

Reply:
Moved to a table.